platform/logging/appenders/file/FileAppenderConfig.ts

```diff
@@ -0,0 +1,33 @@
+import * as schema from '../../../lib/schema';
+import { BaseAppenderConfig, getRawSchema as getBaseRawSchema } from '../base/BaseAppenderConfig'
+import { typeOfSchema } from '../../../types';
+import { FileAppender } from './FileAppender';
+
+function getSchema() {
+  return schema.object({
+    ...getBaseRawSchema(),
+    kind: schema.literal('file'),
+    path: schema.string()
+  });
+}
+
+const schemaType = typeOfSchema(getSchema);
+type FileAppenderConfigSchemaType = typeof schemaType;
+
+export class FileAppenderConfig extends BaseAppenderConfig {
+  readonly path: string;
+
+  constructor(schema: FileAppenderConfigSchemaType) {
+    super(schema);
+
+    this.path = schema.path;
+  }
+
+  createAppender(): FileAppender {
```

Comment:
This "feels" wrong to me (that the config "owns" creating the appender). Let's chat about it on zoom, I can def be convinced the other way